### ISSUE TITLE
How should the radio buttons that select calculations be set up, it is currently terrible.

### ISSUE THREAD

**Issue by StevenPG** [PhysCal/PhysCal.github.io#16]
Comment ideas below.

**Comment by PhysCal** [comment]
The only big idea I had for this was to essentially rotate our current design of the page 90 degrees. 

The main issue with the buttons is how clustered together they are due to the way the page progresses ;they have to be horizontally next to each other because the actual calculation progresses downwards. So the way I see to rectify this is to make the calculation progress from left to right instead and thus have the buttons spread vertically along the page.

ex. Variable selection
[ ] Initial Velocity
[ ] Final Velocity
[ ] Acceleration                                                          ===>         Boxes for variable values/result box
[ ] Distance
[ ] etc

This would obviously require a bit of reworking formattingly, but in my head it's aesthetically pleasing and neat. The biggest drawback to this I think would be that I would want the other sections of the page to appear sequentially after the previous one has been fulfilled, otherwise the whole page might look weird or dumb. Like the value entry boxes don't show up at all until a variable to calculate has been chosen.

tldr; This is an idea that I think could fix the dumbness of the selection boxes, but might fuck up the aesthetic of the rest of the page as a result.

**Comment by StevenPG** [comment]
I think that would work, what we actually have right now is that sort of thing. 
"The biggest drawback to this I think would be that I would want the other sections of the page to appear sequentially after the previous one has been fulfilled, otherwise the whole page might look weird or dumb"
Thats effectively what is happening now, feel free to correct me/clarify.

**Comment by StevenPG** [comment]
So I attempted to write it that way, and it wont fit on the page. So then I tried using some animations to make the page disappear in seperate pieces, but it falls apart with the aesthetic of the page. We would need to use JQuery which is overly complicated for this scale of a project where visual effect isn't the main goal. I've altered it now, they seem a little bit spaced out but are functionally better, I'm going to rework the formatting a little bit before I commit the files. But this is basically how it is going to look unless there is a new "great" idea.
